src/plugins/presentation_panel/public/panel_actions/edit_panel_action/edit_panel_action.ts

```diff
@@ -45,7 +45,7 @@ export class EditPanelAction
     return i18n.translate('presentationPanel.action.editPanel.displayName', {
       defaultMessage: 'Edit {value}',
       values: {
-        value: embeddable.getTypeDisplayName(),
+        value: embeddable.getTypeDisplayNameLowerCase?.() ?? embeddable.getTypeDisplayName(),
```

Comment:
See my comment about why I don't think `getTypeDisplayNameLowerCase` is necessary